cves/kernel/CVE-2016-3137.yml

```diff
@@ -425,8 +424,8 @@ lessons:
     applies:
     note:
   complex_inputs:
-    applies:
-    note:
+    applies: true
+    note: 'In a way this vulnerability is due to complex inputs. The provided serial device is the input in this case and the complexity of the inputs available on such a device is astronomical. This vulnerability is due to impropper handling of this input.'
```

Comment:
comma after In a way (i.e. In a way, this vulnerability...) Typo in improper, you have "impropper."

Reply:
Fixed improper but I removed "In a way all together, it felt wishy-washy.